src/tls.cpp

```diff
@@ -166,6 +166,11 @@ TLSInfo_GetTidOffset(void)
   }
 #endif
 
+#ifdef __riscv
```

Comment:
There is a later commit in the branch `feature/riscv-experimental`.
Please check that these two commits are doing the same thing.  We'll need only one of them.